rollup.config.js

```diff
@@ -47,5 +48,6 @@ export default {
       ENV: JSON.stringify(process.env.NODE_ENV || 'development'),
     }),
     (process.env.NODE_ENV === 'production' && uglify()),
+    (process.argv.indexOf('--live') !== -1 && livereload('build')),
```

Comment:
```js
(process.argv.includes('--live') && livereload('build')),
```

I am pretty sure you can use `includes` for these purposes in most recent versions of node... am I wrong?